packages/react-graphql/CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## [Unreleased] -->
+## [Unreleased]
+
+- Add `fetchMore` param to `useQuery` hook result documention ([#1398](https://github.com/Shopify/quilt/pull/1398)).
```

Comment:
I don't think you need this because we will not release a new version for doc changes.